### DISCUSSION TITLE
Strecken der tramAachen

### DISCUSSION THREAD

**Discussion by domib97** [domib97/tramAachen#2] [Ideas]
1. Aachen Nord über AC West nach Aachen Süd
2. Ring-Route
3.

**Comment by FelixBeckmann** [comment]
Hi, ich finde das Projekt interessant weil ich aus einer Stadt mit gutem Tram-Netz komme und die Busse in Aachen wirklich sehr zu wünschen übrig lassen. Wollte dich nur mal fragen wie du die Zukunft für dieses Projekt siehst /  was du weiter damit vor hast / inwiefern du Chancen auf Umsetzung siehst. Ich würde mich sehr gern daran beteiligen. Freue mich auf deine Antwort.

LG Felix :)

**Reply by domib97** [reply]
Hi Felix! Also die nächsten Schritte wären Mitstreiter zu finden und die groben Aufgabenfelder zu definieren. Und mit genug Menschen und Sponsoren könnte man das Projekt dann in ein Start Up/Firma umwandeln. Also ich wäre über deine Hilfe egal in welcher Form sehr dankbar! Hättest du Ideen und Vorschläge wie man das umsetzen könnte?
